codespell_lib/tests/test_dictionary.py

```diff
@@ -17,6 +17,7 @@
     for lang in supported_languages:
         spellers[lang] = aspell.Speller('lang', lang)
 except Exception as exp:  # probably ImportError, but maybe also language
+    spellers = dict()
```

Comment:
Might be better to have a single `spellers = dict()` line above the `try:` statement?

Reply:
done